plugins/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStore.java

```diff
@@ -241,163 +246,57 @@ void deleteBlobsByPrefix(String prefix) throws IOException {
      * @param blobNames names of the bucket to delete
      */
     void deleteBlobs(Collection<String> blobNames) throws IOException {
-        if (blobNames == null || blobNames.isEmpty()) {
+        if ((blobNames == null) || blobNames.isEmpty()) {
```

Comment:
Note: There's no strict rule but we don't always add parenthesis in `if` conditions. In this case for example, it does not bring much.